source/_integrations/konnected.markdown

```diff
@@ -32,7 +32,11 @@ Konnected devices communicate with Home Assistant over your local LAN -- there i
 
 ### Configuration
 
-A `konnected` section must be present in the `configuration.yaml` file that specifies the Konnected devices on the network and the sensors or actuators attached to them:
+Home Assistant offers Konnected integration through **Configuration** -> **Integrations** -> **Konnected.io**.
+
+The config flow will guide you through a setup process that lets you configure the behavior of each zone and generate a config entry. 
```

Comment:
Please check and update this section after the changes in the parent PR regarding the config and options flow.

Reply:
on it